### ISSUE TITLE
[FR]: Pitch shift, time stretch using SignalSmith/Bungee

### ISSUE THREAD

**Issue by toaster217** [Tracktion/tracktion_engine#229]
SignalSmith and Bungee are two outstanding libraries with permissive licenses. I hope tracktion can add support for these two, preferably bungee since it has great performance

**Comment by drowaudio** [comment]
Not against this but it would be a chunk of work to do. If someone wants to make a decent start and offer a PR that would help expedite it.

I had a quick look at Bungee several months ago but thought the really good one (Pro I think they call it) is a paid-for product?

**Comment by toaster217** [comment]
Yes Pro version is the paid one, which has better audio quality, but I think the free version is already nice.

**Comment by drowaudio** [comment]
We don't have bandwidth to add that at the moment but like I said if someone wants to start it off with a decent PR that would help 😉
